cli/cli_set_technology.go

```diff
@@ -69,7 +69,6 @@ func (c *cmd) SetTechnology(ctx *cli.Context) error {
 		flag, _ := strconv.ParseBool(resp.Data[0])
 		color.Green(fmt.Sprintf(MsgSetSuccess, "Technology", strings.Join(resp.Data[1:], " ")))
 		c.config.Technology = tech
-		c.config.Obfuscate = false
```

Comment:
The application uses this bool also for `Nordlynx`. Not setting it to false will break the commands that are using it.
 e.g.: `nordvpn countries`, `nordvpn cities Germany`